code/modules/crafting/recipes.dm

```diff
@@ -33,17 +33,20 @@
 	category = CAT_WEAPONRY
 	subcategory = CAT_WEAPON
 
+/* - yogs change: removes explosive lance for being stupid
 /datum/crafting_recipe/lance
 	name = "Explosive Lance (Grenade)"
 	result = /obj/item/twohanded/spear
 	reqs = list(/obj/item/twohanded/spear = 1,
 				/obj/item/grenade = 1)
-	blacklist = list(/obj/item/twohanded/spear/explosive)
+	blacklist = list(/obj/item/twohanded/spear/explosive,
+					/obj/item/grenade/flashbang) //yogs change - removes flashbangs from explosive lances for being even stupider
```

Comment:
Are you trying to remove lances or blacklist flashbangs?

Reply:
i'm expecting some random idiot to swing by later and uncomment this, and if not, i'm going to add explosive  lances back anyway, minus explosive grenades

but lance flashbangs need to go anyway, so they're getting blacklisted so that when i finally comment this back in they still can't be made